kotlin-android/src/test/java/com/segment/analytics/kotlin/android/utils/Mocks.kt

```diff
@@ -0,0 +1,35 @@
+package com.segment.analytics.kotlin.android.utils
+
+import android.content.Context
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import com.segment.analytics.kotlin.android.utils.MemorySharedPreferences
+import com.segment.analytics.kotlin.core.Analytics
+import io.mockk.every
+import io.mockk.mockk
+import sovran.kotlin.Store
+import java.io.File
+
+fun mockAnalytics(): Analytics {
```

Comment:
<documentation>? It may be beneficial to add here for when we have new eng coming and knowing if / who they should use this for testing.